src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -415,7 +415,26 @@ public virtual bool RunInMemory
         [IndexUpdateType(IndexUpdateType.None)]
         [ConfigurationEntry("Indexing.Static.RequireAdminToDeployJavaScriptIndexes", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public bool RequireAdminToDeployJavaScriptIndexes { get; set; }
-
+        
+        [Description("Order by score automatically when boosting is inside index definition or query.")]
+        [DefaultValue(true)]
+        [IndexUpdateType(IndexUpdateType.Refresh)]
+        [ConfigurationEntry("Indexing.OrderByScoreAutomaticallyWhenBoostingIsInvolved", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
+        public bool OrderByScoreAutomaticallyWhenBoostingIsInvolved { get; set; }
+        
+        [Description("EXPERT: Use compound file in merging")]
+        [DefaultValue(true)]
```

Comment:
Note that we're not changing the current option. Currently it does use compound files as well.

For reference:
https://lucene.apache.org/core/3_0_3/fileformats.html

Reply:
So I wonder if we shouldn't disable that by default

Reply:
Maciej is gonna do one more test to check if the usage of compound file has any effect on query timings

Reply:
Yes, I would expect to set that to `false` by default.

Reply:
We got even slightly better querying results with compound file turned on. Let's leave it as the default - same as it was before.